packages/server/.env.example

```diff
@@ -3,7 +3,7 @@
 # Instance Configuration:
 ARCWELL_INSTANCE_NAME="Development Arcwell Server"
 ARCWELL_INSTANCE_ID=arcweb-dev
-ARCWELL_SERVER_CORS_ORIGINS=http://localhost:4200,0.0.0.0:4200,127.0.0.1:4200,http://localhost:3333
+ARCWELL_SERVER_CORS_ORIGINS=http://localhost:4200,0.0.0.0:4200,127.0.0.1:4200,http://localhost:3333,http://localhost:4000,capacitor://localhost,http://localhost
```

Comment:
@Alex-Smith-Arcweb this appears to be related to the _simulator_ during local development.. NOT the app running on someone's phone. Can you confirm this?

Reply:
Sorry to keep quibbling, but A person's phone in their hand has its own IP address and hostname that is 100% NOT `localhost` when received by the Arcwell Server _except in the case where it's running in the simulator, CI, or similar_.

It's unclear to me how these entries would even work for a compiled app use case.

Reply:
But I'm finding other docs online that say "just add localhost" to your CORS origins. Do we know why or have any reference from Capacitor itself?

Reply:
@Alex-Smith-Arcweb shared this which starts to explain it: https://forum.ionicframework.com/t/how-works-cors-in-capasitor-on-real-devices/230474 👍